### PR TITLE
 Dependabot PRs - time box 9PM -9AM EST 

### PR DESCRIPTION
**Pull request by bethbeza**
Two workflows were added to manage Dependabot's schedule:

- [x] - Disable Dependabot at 9 AM EST to pause updates during work hours.
- [x] - Enable Dependabot at 9 PM EST to resume updates outside work hours.
- [ ] - Ensure Dependabot can create, approve and merge its own PRs

### CODE REVIEW

File: ting signed commit"
Comment (devopsmatt) [inline ting signed commit":1]: Please remove this file from the PR.

File: test-signed.txt
Comment (devopsmatt) [inline test-signed.txt:1]: Please remove this file from the PR.

File: ting signed commit"
Comment (bethbeza) [reply ting signed commit":1]: this item is now removed.

File: test-signed.txt
Comment (bethbeza) [reply test-signed.txt:1]: It's now removed.

**Comment by bethbeza** [comment]
fixing unsigned commits with team  mate.

**Review by scott-aquia — COMMENTED** [review]
I have reviewed these changes from a security perspective

File: dependabot-scheduler.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: ${{ secrets.PAT_TOKEN }}
+          branch: update-branch
+          title: "Automated Dependency Updates"
+          body: "Automated dependency updates from GitHub Actions."
+          base: main
+          commit-message: "Update dependencies"
+
```
Comment (devopsmatt) [inline dependabot-scheduler.yml:35]: Add GHA approval on the PR here

**Review by scott-aquia — COMMENTED** [review]
I have reviewed this change from a security perspective

**Pull request closed without merging**